src/hooks/useDrag.tsx

```diff
@@ -333,7 +396,35 @@ const updateHomepageState = (
         announcementsIndex
       )
     }
+    case "textcardcard": {
+      const parentId = parseInt(typeArray[1], RADIX_PARSE_INT)
+      const draggedElem = ((frontMatter.sections[
+        parentId
+      ] as TextcardFrontmatterSection).textcards as EditorTextcardSection)
+        .cards[source.index]
```

Comment:
would it be possible to just cast once at the top so it's easier to follow?

Reply:
Good catch,  https://github.com/isomerpages/isomercms-frontend/pull/1505/commits/215a1ff08d571782ed4873b7fa16efd23343088a